nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/WorkflowEndpointTest.java

```diff
@@ -560,13 +563,24 @@ public void testDelegateTask() throws IOException {
 
         // Delegate
         taskId = getCurrentTaskId(createdWorkflowInstanceId);
+        List<String> delegatedActors = Arrays.asList("members", "Administrator");
         MultivaluedMap<String, String> queryParams = new MultivaluedMapImpl();
-        queryParams.put("actors", singletonList("members"));
+        queryParams.put("delegatedActors", delegatedActors);
```

Comment:
Are the actors and delegatedActors exclusive?

Reply:
No, they are not. In fact, the first one "actors" is not considered by the delegate endpoint.

https://github.com/nuxeo/nuxeo-platform-document-routing/blob/094ce7bf5fc08ecae34119a95ac7786a6dc29b1c/nuxeo-routing-rest-api/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/routing/TaskObject.java#L69